_component_design/tables-nested.md

```diff
@@ -0,0 +1,114 @@
+preview-image: preview-images/tables-nested.svg
+resource: true
+last-modified: 2017-08-17
+---
+
+{% include toc.html %}
+
+## Introduction
+{: .hxSectionTitle}
+<div class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### When to use
+{: .hxSubSectionTitle}
+Nested tables allow the nesting of additional levels of data within a table.
+
+
+### Best practice
+{: .hxSubSectionTitle}
+
+- All rows should be closed by default.
+- The expanded area inherits the styling of the parent row. For example, bound or unbound, and zebra stripe color.
+- There may only be one expandable content area per row.
+- Expand may be triggered in three ways:
+	- Clicking anywhere within the row that is not a link.
+	- Clicking the down-carat icon.
+	- Clicking a badge icon.
+- Collapse may be triggered in two ways:
+	- Clicking anywhere in the parent row.
+	- Clicking directly on the collapse icon (either in a parent row or at the bottom of an expanded area).
+-  Rows remain expanded until collapsed by user.
+-  Include a collapse icon in the bottom right corner of the expanded area in addition to the collapse icon in the parent row.
+-  Viewport remains unchanged with expansion and collapse.
+
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Meet the nested table pattern"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/tables/nested-tables/tables-nested-hero.svg){:width="100%"}
+{% endfigure %}
+{% figure [caption:"Nested table IxD"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/tables/nested-tables/tables-nested-ixd.svg){:width="100%"}
+{% endfigure %}
+{% endcolumn %}
+
+</div>
+
+## Variations
+{: .hxSubSectionTitle}
+
+<div class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+### Explanatory text variation
+{: .hxSubSectionTitle}
+
+This variation of an expandable table is used to add additional supporting text to a table row.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Nested table IxD"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/tables/nested-tables/tables-explanatory.svg){:width="100%"}
+{% endfigure %}
+{% endcolumn %}
+
+</div>
+
+<div class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Key:value variation
+{: .hxSubSectionTitle}
+
+This variation of an expandable table is used to add additional key value pairs of associated meta data to a table row.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Key:Value expandable row. NOT TO SCALE"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/tables/nested-tables/tables-key-value.svg){:width="100%"}
```

Comment:
This image will need to change to a vertical key-value list when we relaunch the site, but this is ok for now since Becka is wrapping up that work.

Reply:
noted for later. Also I removed all instances of `{:width="100%"}` across the three docs because I saw it here and realized you hadn't yet requested its removal =)